modules/options/src/main/scala/scala/build/actionable/ActionablePreprocessor.scala

```diff
@@ -0,0 +1,21 @@
+package scala.build.actionable
+
+import scala.build.Ops._
+import scala.build.errors.{BuildException, CompositeBuildException}
+import scala.build.options.BuildOptions
+
+case object ActionablePreprocessor {
```

Comment:
This shouldn't be a case object, as this isn't meant to be passed around as data.
```suggestion
object ActionablePreprocessor {
```